chef/cookbooks/barclamp/libraries/nic.rb

```diff
@@ -566,7 +566,6 @@ def add_slave(slave)
       usurp(slave)
```

Comment:
I'm slightly confused here: with the `ifdown` call that is being done, don't we kill the `usurp` call here?